### ISSUE TITLE
Enable cron builds in Travis CI

### ISSUE THREAD

**Issue by webknjaz** [pengutronix/aiohttp-json-rpc#26]
@fscherf hi, there's a few improvements to make, CI-wise:
1) Run tests against `master` on daily basis
2) Migrate integration into new platform travis-ci.com

**Comment by fscherf** [comment]
@webknjaz: why do we need to test daily? Because of aiohttp updates?

**Comment by webknjaz** [comment]
Exactly

**Comment by fscherf** [comment]
@webknjaz whats the difference between travis-ci.org and travis-ci.com?

**Comment by webknjaz** [comment]
yes, it's possible to do it this way and also it's possible to do a conditional stage runs, for example, only test  forward compatibility within cron jobs.

travis-ci.com is used to be for paid customers only, but now they're merging two platforms into one. So you'll basically get the same. Sooner or later they'll migrate you anyway. Currently you can opt-in for immediate migration, but this won't migrate your old build history (it will be available in the old web-site during some time).
Also, they are going away from the deprecated GitHub Services API towards newer Apps API, which also makes it much more flexible for you to control Travis's access to your repos.

**Comment by webknjaz** [comment]
More info:
https://blog.github.com/2018-05-07-introducing-checks-api/
https://blog.travis-ci.com/2018-05-07-announcing-support-for-github-checks-api-on-travis-ci-com
https://developer.github.com/v3/checks/

Oh, and btw I've already migrated all repos in aio-libs org to the new platform.

**Comment by fscherf** [comment]
Sounds good to me. Thanks for bringing this up! Would be great if you send patches :)

**Comment by webknjaz** [comment]
Well, you can already ask Travis' support to migrate the repo and you can already enable cron builds. Both do not depend on patches.
